lib/modules/manager/bundler/common.ts

```diff
@@ -34,17 +34,24 @@ export async function getRubyConstraint(
       logger.debug('Using ruby version from gemfile');
       return rubyMatch;
     }
-    const rubyVersionFile = getSiblingFileName(
-      packageFileName,
-      '.ruby-version',
-    );
-    const rubyVersionFileContent = await readLocalFile(rubyVersionFile, 'utf8');
-    if (rubyVersionFileContent) {
-      logger.debug('Using ruby version specified in .ruby-version');
-      return rubyVersionFileContent
-        .replace(regEx(/^ruby-/), '')
```

Comment:
I don't know if there is (still) `.ruby-version` that contains a ruby version prefixed with `ruby-`, but I kept the support for it anyway (as in line 40 below) not to break any existing projects.